lib/xref.h

```diff
@@ -158,17 +158,18 @@ extern const struct xref * const __stop_xref_array[1] DSO_LOCAL;
  */
 #define XREF_SETUP()                                                           \
 	static const struct xref _dummy_xref = {                               \
-			/* .xrefdata = */ NULL,                                \
-			/* .type = */ XREFT_NONE,                              \
-			/* .line = */ __LINE__,                                \
-			/* .file = */ __FILE__,                                \
-			/* .func = */ "dummy",                                 \
+		/* .xrefdata = */ NULL,                                        \
+		/* .type = */ XREFT_NONE,                                      \
+		/* .line = */ __LINE__,                                        \
+		/* .file = */ __FILE__,                                        \
+		/* .func = */ "dummy",                                         \
 	};                                                                     \
-	static const struct xref * const _dummy_xref_p                         \
-			__attribute__((xref_array_attr)) = &_dummy_xref;       \
-	static void __attribute__((used, _CONSTRUCTOR(1100)))                  \
-			_xref_init(void) {                                     \
+	static const struct xref *const _dummy_xref_p                          \
+		__attribute__((xref_array_attr)) = &_dummy_xref;               \
+	static void __attribute__((used, _CONSTRUCTOR(1100))) _xref_init(void) \
+	{                                                                      \
```

Comment:
nit: remove whitespace changes or move them to a separate commit

Reply:
I had to give up on emacs and use vi to avoid it trying to "fix" the indention.. so odd.